setup.py

```diff
@@ -27,11 +27,13 @@
     url="https://github.com/pvizeli/pycognito",
     license="Apache License 2.0",
     install_requires=[
-        "boto3>=1.10.49",
```

Comment:
that is only true for lambda maybe, but all use case that I use in this library is far away from running inside a lambda. I can't approve of this breaking change. If you would make a solution that work the other way around, I'm fine to approve it

Reply:
When you say "make a solution that work the other way around," do you mean to have a requirement dependency that could be removed? As far as I know, the best I could do is to define it as an [optional dependency](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies), but that would also be a breaking change in your context.

Do you have a mechanism in mind for what you're thinking?

Reply:
As this is imported and used in the code, this needs to be present under `install_requires` (default behavior).
For that reason, Im going to close this PR as a change like this can not be accepted unless the library is changed to not rely on that package at all.
Thanks anyway 👍 

For your packaging you can look at something like <https://blog.frank-mich.com/minimize-the-size-of-lambdas-by-deleting-boto3-from-the-deployment-package/> instead.